Javatrix.java

```diff
@@ -22,6 +22,37 @@
     private int m;
     private int n;
 
+     /**
+     * Constructor 1D array.
+     *
+     * @param m number of rows
+     * @param vals stored by columns
+     * @throws IllegalArgmentException if length of 
+     * val is not a multiple of m
```

Comment:
Argument is misspelled here after the @throws tag.  That's why check style is complaining